cmd/secrets-provider/main.go

```diff
@@ -51,51 +52,55 @@ func main() {
 		printErrorAndExit(messages.CSPFK009E)
 	}
 
-	// Configure exponential backoff
-	expBackoff := backoff.NewExponentialBackOff()
-	expBackoff.InitialInterval = 2 * time.Second
-	expBackoff.RandomizationFactor = 0.5
-	expBackoff.Multiplier = 2
-	expBackoff.MaxInterval = 15 * time.Second
-	expBackoff.MaxElapsedTime = 2 * time.Minute
+	syncIntervalTime := 3
```

Comment:
do we want to have this as some costant?

Reply:
Yes, this is a default

Reply:
added to consts